Telegram/SourceFiles/kotato/settings.cpp

```diff
@@ -64,6 +64,17 @@ rpl::producer<bool> AdaptiveBubblesChanges() {
 	return gAdaptiveBubbles.changes();
 }
 
+rpl::variable<bool> gMonospaceLargeBubbles = false;
+void SetMonospaceLargeBubbles(bool enabled) {
+	gMonospaceLargeBubbles = enabled;
+}
+bool MonospaceLargeBubbles() {
+	return gMonospaceLargeBubbles.current();
+}
+rpl::producer<bool> MonospaceLargeBubblesChanges() {
+	return gMonospaceLargeBubbles.changes();
+}
+
```

Comment:
It seems you're using `rpl::variable`, but I don't see any code to refresh it on the fly. Though it should be easy since you can reuse same code as in AdaptiveBubblesChanges:

https://github.com/kotatogram/kotatogram-desktop/blob/d0fd79001d798b1141202d69fab3b7413d137781/Telegram/SourceFiles/history/history_widget.cpp#L532-L544

Reply:
Examples of `Ui::SlideWrap` usage:

https://github.com/kotatogram/kotatogram-desktop/blob/f9ad9ce03bba67845e65e14295dbe8a098afcadb/Telegram/SourceFiles/settings/settings_notifications.cpp#L557-L561

https://github.com/kotatogram/kotatogram-desktop/blob/f9ad9ce03bba67845e65e14295dbe8a098afcadb/Telegram/SourceFiles/settings/settings_notifications.cpp#L566-L571

https://github.com/kotatogram/kotatogram-desktop/blob/f9ad9ce03bba67845e65e14295dbe8a098afcadb/Telegram/SourceFiles/settings/settings_notifications.cpp#L666-L668